src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
These imports (java.util.Map, java.util.Objects, java.util.Optional, java.util.stream.Collectors, java.util.stream.Stream) are not used in the current class. It's a good practice to remove unused imports to keep the code clean.